### PR TITLE
Feature/KITOSUDV-3212-merge-fix

### PR DESCRIPTION
**Pull request by a-naskret**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [ ] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/app/components/it-system/it-system-overview.controller.ts [hunk 1]
```diff
@@ -477,36 +477,14 @@
...
-                            if (dataItem.MainContractIsActive == null) {
-                                return "";
-                            }
-                            const decorationClass = dataItem.MainContractIsActive
-                                ? "fa-file text-success"
-                                : "fa-file-o text-muted";
-                            return `<a data-ui-sref="it-system.usage.contracts({id: ${dataItem.SourceEntityId}})"><span class="fa ${decorationClass}" aria-hidden="true"></span></a>`;
-                        })
+                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderActiveNotActive(dataItem.MainContractIsActive))
                         .withExcelOutput(dataItem => dataItem.MainContractIsActive ? "True" : "")
```
Comment (a-naskret) [inline Presentation.Web/app/components/it-system/it-system-overview.controller.ts:488]: should the excel value stay the same?

**Comment by a-naskret** [comment]
@mrjsawdk ready for review

**Comment by mrjsawdk** [comment]
@a-naskret I merged master into your branch

**Review by mrjsawdk — APPROVED** [review]
@a-naskret merge when green after master merge: https://teamcity.strongminds.dk/viewLog.html?buildId=9871&tab=buildResultsDiv&buildTypeId=Os2Kitos_DeployToIntegration

**Pull request merged**